toolchain/sem_ir/inst.h

```diff
@@ -462,20 +462,21 @@ class InstBlockStore : public BlockValueStore<InstBlockId> {
     CARBON_CHECK(global_init_id == InstBlockId::GlobalInit);
   }
 
-  // Reserves and returns a block ID. The contents of the block should be
-  // specified by calling Set, or similar.
-  auto AddPlaceholder() -> InstBlockId {
-    return values().Add(llvm::MutableArrayRef<ElementType>());
-  }
-
   // Adds an uninitialized block of the given size. The caller is expected to
   // modify values.
   auto AddUninitialized(size_t size) -> InstBlockId {
     return values().Add(AllocateUninitialized(size));
   }
 
+  // Reserves and returns a block ID. The contents of the block should be
+  // specified by calling ReplacePlaceholder.
+  auto AddPlaceholder() -> InstBlockId {
+    return values().Add(llvm::MutableArrayRef<ElementType>());
+  }
+
   // Sets the contents of an empty block to the given content.
```

Comment:
```suggestion
  // Sets the contents of a placeholder block to the given content.
```
Just to avoid the suggestion that you can `Add({})` and then `ReplacePlaceholder`. Might also be nice to CHECK-fail if `block_id` is `InstBlockId::Empty`.

Reply:
Done